blob/service.go

```diff
@@ -216,13 +217,14 @@ func (s *Service) Included(
 	return true, resProof.equal(*proof)
 }
 
-// getByCommitment retrieves the DAH row by row, fetching shares and constructing blobs in order to
+// retrieve retrieves the DAH row by row, fetching shares and constructing blobs in order to
```

Comment:
This comment is confusing. Retrieve does not retrieve the DAH, but Blob and the Proof.

Reply:
It retrieves the namespace though. Let's change DAH to namespace, wdyt?

Reply:
No, it retrieves the blob and the proof. It does so by retrieving the whole namespace and thats an implementation detail that we can mention

Reply:
The whole method is an implementation detail as it's unexpected and is not part of the API 🤣 . But, ok. will change the description

Reply:
It's part of the private API